utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -92,6 +92,12 @@ fun Double.format(decimalPlaces: Int = 2) = "%.${decimalPlaces}f".format(this)
  */
 operator fun <E : Enum<E>> EnumSet<E>.plus(other: EnumSet<E>): EnumSet<E> = EnumSet.copyOf(this).apply { addAll(other) }
 
+/**
+ * Return the path to this file or a corresponding message if the file is unknown.
+ */
+val File?.safePath: String
```

Comment:
This is used only in a very narrow context. But it now will appear in the File API globally.
While I agree that it is similar to `safeMkdirs()` soley by name, I doubt that is in other ways similar which matter more. 
I'd prefer to keep this module internal.

Reply:
Fine. I've dropped that commit to proceed.